pyaedt/modules/report_templates.py

```diff
@@ -3786,3 +3786,220 @@ def create(self, plot_name=None):
         self._post.plots.append(self)
         self._is_created = True
         return True
+
+
+class EMIReceiver(CommonReport):
+    """Provides for managing EMI receiver reports."""
+
+    def __init__(self, app, setup_name, expressions=None):
+        CommonReport.__init__(self, app, "EMIReceiver", setup_name, expressions)
+        self.logger = app.logger
+        self.domain = "EMI Receiver"
+        self.available_nets = []
+        self._net = "0"
+        for comp in app.modeler.components.components.values():
+            if comp.name == "CompInst@EMI_RCVR":
+                self.available_nets.append(comp.pins[0].net)
+        if self.available_nets:
+            self._net = self.available_nets[0]
+        self.time_start = "0ns"
+        self.time_stop = "200ns"
+        self._emission = "CE"
+        self.overlap_rate = 95
+        self.band = "0"
+        self.primary_sweep = "Freq"
+
+    @property
+    def net(self):
+        """Net attached to EMI receiver.
+
+        Returns
+        -------
+        str
+        """
+        return self._net
+
+    @net.setter
+    def net(self, value):
+        if value not in self.available_nets:
+            self.logger.error("Net not available.")
+        else:
+            self._net = value
+
+    @property
+    def band(self):
+        """Net attached to EMI receiver.
```

Comment:
```suggestion
        """Net attached to the EMI receiver.
```
Should "Net" be "Band"?